### PR TITLE
qmckl_compute_asymp_jasb_f to C

### PR DESCRIPTION
**Pull request by GianFree**
Let me know whether you prefer a PR per function, or to group several of them into a single request.

### CODE REVIEW

**Comment by v1j4y** [comment]
> Let me know whether you prefer a PR per function, or to group several of them into a single request.

I would prefer as few PRs as possible since that saves review time. This would also help unforeseen changes encounter at a later date.

File: org/qmckl_jastrow.org [hunk 1]
```diff
@@ -1304,42 +1303,54 @@ integer function qmckl_compute_asymp_jasb_f(context, bord_num, bord_vector, resc
...
-	bind(C) result(info)
+    if (context == QMCKL_NULL_CONTEXT){
+      return QMCKL_INVALID_CONTEXT;
+    }
 
-      use, intrinsic :: iso_c_binding
-      implicit none
+    if (bord_num <= 0) {
+      return QMCKL_INVALID_ARG_2;
+    }
```
Comment (v1j4y) [inline org/qmckl_jastrow.org:1324]: Nice that you put the checks for the arguments !

**Review by v1j4y — COMMENTED** [review]
Looks good up to now. 

Would you like to merge this or keep pushing to this PR (with a name change) for other functions also ?

File: org/qmckl_jastrow.org (hunk 1 above)
Comment (GianFree) [reply org/qmckl_jastrow.org:1324]: Thanks for the credits, but the checks were already in place, so kudos to you!

> Would you like to merge this or keep pushing to this PR (with a name change) for other functions also ?

For me it's the same. If it is less annoying for you, just use this one and I'll add commit to this.

Comment (v1j4y) [reply org/qmckl_jastrow.org:1324]: I'll merge this for now.
Maybe for the next time it would be better if you send one PR for the whole Jastrow module.

**Pull request merged**